examples/graphene_apps/python_worker/tests/test_ov_work_orders.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Is it a custom data format that the openvino work load using? If yes add comment.

Reply:
We can't add comments in json file. Format is mentioned in ReadMe https://github.com/hyperledger/avalon/tree/master/examples/graphene_apps/python_worker#adding-a-new-python-workload